docs/admin/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Administration guide"
+weight: 10
+type: "docs"
+---
+
+There are two core Knative components that can be installed and used together or independently to provide different functions:
+
+* [Knative Serving](../serving/): Easily manage stateless services on Kubernetes by reducing the developer effort required for autoscaling, networking, and rollouts.
+
+* [Knative Eventing](../eventing/): Easily route events between on-cluster and off-cluster components by exposing event routing as configuration rather than embedded in code.
+
+These components are delivered as Kubernetes custom resource definitions (CRDs), which can be configured by a cluster administrator to provide default settings for developer-created applications and event workflow components.
```

Comment:
I think there's also some common configuration options for administrators around telemetry (`config-logging`, `config-observability`) that should probably move to the admin guide:

https://knative.dev/docs/install/collecting-metrics/
https://knative.dev/docs/install/collecting-logs/

(I'm not quite sure what to do with the "here's a local-logs-collection solution" at the end of the second; probably turn it into something real or go back and delete it.)

Reply:
Nevermind, it looks like you did this. Do you want to mention "common configuration options" as another section here, or just rely on the left-hand nav?

Reply:
I don't want to add a bunch of links to maintain really until things settle down a bit with moving stuff, but for now I've added the contents list back to the page; does that work for you?